app/adaptive_tessellation/Split.cpp

```diff
@@ -15,6 +15,35 @@ auto split_renew = [](auto& m, auto op, auto& tris) {
     return optup;
 };
 
+double AdaptiveTessellation::get_quadrics_area_accuracy_error_for_split(
```

Comment:
Where are we computing the error _after_ the split here? Do we even care? Because if it's only for the "priority", I would simply use the sum of the quadrics for the two vertices involved in the split, rather than all 2 vertices around an edge.

Reply:
The update happens here ([(https://github.com/wildmeshing/wildmeshing-toolkit/blob/b078116ca65f239e06a640aa3b5d760e7512e6ad/app/adaptive_tessellation/Split.cpp#L207-L224)])